ice/diagnostic_requests/ice_history_detailed.nml

```diff
@@ -0,0 +1,128 @@
+&setup_nml
+  , histfreq       = 'd','m','x','x','x'
+  , histfreq_n     = 1, 1, 1, 1, 1
+  , hist_avg       = .true.
+  , history_file   = 'iceh'
+  , history_format = 'nc'
```

Comment:
We'll never change these two lines, so I think leave them in `cice_in.nml`

Reply:
Ah that's good to know! I've moved these two back into `cice_in.nml`